.github/scripts/run-linux-poweroff-vitis.py

```diff
@@ -30,6 +30,11 @@ def run_linux_poweroff_vitis():
 
                 run("./marshal -v install test/outputs.yaml")
 
+            # download prebuilt xclbin to /tmp
+            # this CI image is managed by the default CI account
+            with prefix('cd /tmp'):
```

Comment:
Did you upload this manually? Would be interesting to get rocket builds in CI and upload them, and then query this bucket to download. Out of scope for this PR, but could be a stretch goal down the line.

Reply:
Yea that's been a goal for a while but we (or I) need to allocate time to it that I don't have currently.

Reply:
Yes, this was uploaded manually. All people that have access to the CI account can access and upload to the bucket.

Reply:
Sounds good. You should also look at #924